sqlglot/dialects/athena.py

```diff
@@ -1,33 +1,27 @@
 from __future__ import annotations
 
-import typing as t
+from typing import Any, Callable, ClassVar
 
 from sqlglot import exp
-from sqlglot.dialects.trino import Trino
 from sqlglot.dialects.hive import Hive
+from sqlglot.dialects.trino import Trino
 from sqlglot.tokens import TokenType
 
 
 def _generate_as_hive(expression: exp.Expression) -> bool:
     if isinstance(expression, exp.Create):
         if expression.kind == "TABLE":
-            properties: t.Optional[exp.Properties] = expression.args.get("properties")
+            properties: exp.Properties | None = expression.args.get("properties")
```

Comment:
Again, please avoid making changes like these. It makes reviewing PRs harder. Stylistic changes need to be discussed with the core team first and, if accepted, reviewed separately.